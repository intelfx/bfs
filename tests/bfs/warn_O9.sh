# Regression test: don't crash when warning if -O9 is the last argument
cd basic
bfs_diff -warn -O9
