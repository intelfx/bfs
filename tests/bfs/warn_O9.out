.
./a
./b
./c
./c/d
./e
./e/f
./g
./g/h
./i
./j
./j/foo
./k
./k/foo
./k/foo/bar
./l
./l/foo
./l/foo/bar
./l/foo/bar/baz
